stash/src/processing/PriceProcessorService.ts

```diff
@@ -160,26 +157,17 @@ const prepareKusama = async () => {
     return
   }
 
-  const prices = await getCoinHistory(
```

Comment:
Since in eigen-layer-monorepo/stash we only have new rollup-* envs, clauses and data like this are unnecessary and removed.